docs/ref/applications.txt

```diff
@@ -73,9 +73,8 @@ If there is no ``default_app_config``, Django uses the base
 ``django.contrib.admin`` to opt-in to :class:`~django.apps.AppConfig` features
 without requiring users to update their :setting:`INSTALLED_APPS`.
 
-New applications should avoid ``default_app_config``. Instead they should
```

Comment:
does anyone know of any django apps on PyPI that actually force people to use 'package_name.apps.PackageNameAppConfig'?

Reply:
the usual style is:
```
* PyPI: django-app-name
* INSTALLED_APPS: 'app_name'
```